### PR TITLE
test(stack): Update expected results

### PR DESCRIPTION
**Pull request by fviernau**
Also update the `.cabal` file.

### CODE REVIEW

File: plugins/package-managers/stack/src/funTest/assets/projects/synthetic/stack-yesodweb-simple/stack-yesodweb-simple.cabal [hunk 1]
```diff
@@ -1,10 +1,10 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.33.0.
+-- This file has been generated from package.yaml by hpack version 0.37.0.
```
Comment (sschuberth) [inline plugins/package-managers/stack/src/funTest/assets/projects/synthetic/stack-yesodweb-simple/stack-yesodweb-simple.cabal:3]: Commit message:
- "function*al* tests"

File: plugins/package-managers/stack/src/funTest/assets/projects/external/quickcheck-state-machine-expected-output.yml [hunk 2]
```diff
@@ -26,1464 +26,1552 @@ project:
     dependencies: []
   - name: "external"
     dependencies:
-    - id: "Hackage:Control:mtl:2.2.2"
+    - id: "Hackage::QuickCheck:2.13.1"
```
Comment (sschuberth) [inline plugins/package-managers/stack/src/funTest/assets/projects/external/quickcheck-state-machine-expected-output.yml:29]: There's also a Windows version of this file. In the past, I've updated it without actually running the test on Windows by manually taking all updates from the non-Windows file and copying them over while maintaining the Windows-specific parts in the expected output.

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: plugins/package-managers/stack/src/funTest/assets/projects/external/quickcheck-state-machine-expected-output.yml (hunk 2 above)
Comment (sschuberth) [reply plugins/package-managers/stack/src/funTest/assets/projects/external/quickcheck-state-machine-expected-output.yml:29]: Another alternative would be to simply remove *all* Windows-specific expected results:

```
plugins/package-managers/python/src/funTest/assets/projects/external/example-python-flask-expected-output-windows.yml
plugins/package-managers/stack/src/funTest/assets/projects/external/quickcheck-state-machine-expected-output-windows.yml
```

And also simplify the test code accordingly, as no one is running tests regularly on Windows anymore anyway.

Comment (fviernau) [reply plugins/package-managers/stack/src/funTest/assets/projects/external/quickcheck-state-machine-expected-output.yml:29]: I see, I didn't get that your initial comment was a "change request".

So, If I only change as you describe in comment directly above, it's addressed, right ?

Comment (sschuberth) [reply plugins/package-managers/stack/src/funTest/assets/projects/external/quickcheck-state-machine-expected-output.yml:29]: Yes, a preceding commit in this PR that removes the Windows-specific test and data would be fine with me.

File: plugins/package-managers/python/src/funTest/kotlin/PipFunTest.kt [hunk 3]
```diff
@@ -58,8 +57,7 @@ class PipFunTest : WordSpec({
             val definitionFile = getAssetFile("projects/external/example-python-flask/requirements.txt")
 
             // Note: The expected results were generated with Python 3.8 and are incorrect for versions < 3.8.
-            val suffix = "-windows".takeIf { Os.isWindows }.orEmpty()
-            val expectedResultFile = getAssetFile("projects/external/example-python-flask-expected-output$suffix.yml")
+            val expectedResultFile = getAssetFile("projects/external/example-python-flask-expected-output.yml")
```
Comment (sschuberth) [inline plugins/package-managers/python/src/funTest/kotlin/PipFunTest.kt:60]: Commit message:
- typo in "tests tests"

**Review by sschuberth — CHANGES_REQUESTED** [review]

**Review by sschuberth — DISMISSED** [review]

**Comment by sschuberth** [comment]
I've tried you branch locally, and it seems I'd getting the same test failures as on CI. Should I put the results I get into a new PR for checking CI?

**Comment by fviernau** [comment]
> I've tried you branch locally, and it seems I'd getting the same test failures as on CI. Should I put the results I get into a new PR for checking CI?

This would be nice thanks! FYI: I haven't managed to do the copy and pasting approach, so this update came from a local run in docker.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**